lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -34,6 +35,10 @@ define((require, exports, module) ->
     }
 
     initKaTeX = (onLoaded) ->
+      # Loading banned sequences file
+      response = $.getJSON(require.toUrl("./banned_token_sequences.json"))
```

Comment:
do you really need to fetch this list through the request? Why don't you just hardcode it in a structure?